doc/development/guide/installation.rst

```diff
@@ -75,20 +75,6 @@ are required:
 
     conda install psi4 psi4-rt -c psi4
 
-* `Open Babel <https://openbabel.org>`__ (optional but recommended to run the full test suite)
```

Comment:
Great work identifying that we should remove this!

Reply:
Credit goes to @josh146.